data/ExperimentDesignPresets.ts

```diff
@@ -0,0 +1,26 @@
+import {EmptyAAExperiment} from "../types/experiments";
+
+interface Preset<T> {
+  name: string;
+  description: string;
+  presets: Partial<T>
+}
+
+export const AAType: Preset<EmptyAAExperiment> = {
+  name: "A/A Experiment",
+  description: "A design for diagnostic testing of targeting or enrollment. Fixed to 1% of the population.",
+  presets: {
+    branches: [
+      {slug: "control", ratio: 1, value: null},
+      {slug: "treatment", ratio: 1, value: null}
+    ],
+    bucketConfig: {
+      randomizationUnit: "normandy_id",
+      /** This should be unique to the experiment, i.e. generated on the slug */
+      namespace: "",
```

Comment:
I would suggest either omitting this field here altogether or setting its value to null then rather than providing an empty string as a value?

Reply:
Actually, omitting fields is annoying, probably just setting it to `null` is prefered I think.